src/components/create-vm-dialog/pxe-helpers.js

```diff
@@ -136,7 +136,7 @@ export function getPXEInitialNetworkSource(nodeDevices, virtualNetworks) {
     );
 
     if (netNodeDevices.length > 0)
-        return cockpit.format('type=direct,source=$0', netNodeDevices[0].capability.interface);
+        return cockpit.format('type=direct,source=$0,source.mode=bridge', netNodeDevices[0].capability.interface);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1123-20230627-082351-ccab151a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)